webapp-django/crashstats/supersearch/tests/test_views.py

```diff
@@ -851,6 +851,16 @@ def test_get_report_list_parameters(self):
 
         eq_(res['build_id'], ['12345'])
 
+        # Test missing parameters don't break.
+        source = {
+            'date': [None],
+            'product': [None],
+            'version': [None],
+            'release_channel': [None],
+            'build_id': [None],
+        }
+        res = get_report_list_parameters(source)
```

Comment:
If the purpose is to check that it just doesn't raise an error, then don't bother with the `res =`